benchmarks/dbscan/ArborX_DBSCANVerification.hpp

```diff
@@ -47,6 +47,9 @@ bool verifyCorePointsNonnegativeIndex(ExecutionSpace const &exec_space,
         {
 #ifndef __SYCL_DEVICE_ONLY__
           printf("Core point is marked as noise: %d [%d]\n", i, labels(i));
+#else
+          sycl::ext::oneapi::experimental::printf(
+              "Core point is marked as noise: %d [%d]\n", i, labels(i));
 #endif
```

Comment:
Maybe do the following way for all of them?
```suggestion
#ifdef __SYCL_DEVICE_ONLY__
          sycl::ext::oneapi::experimental::
#endif
          printf("Core point is marked as noise: %d [%d]\n", i, labels(i));
```